source/pdk/1.0/pdk_install.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: "Installing the Puppet Development Kit"
+canonical: "/pdk/1.0/pdk_install.html"
+description: "Installing the Puppet Development Kit, the shortest path to developing better Puppet code."
+---
+
+
+**Note: this page is a draft in progress and is neither technically reviewed nor edited. Do not rely on information in this draft.**
+
+Download and install the Puppet Development Kit to get started developing and testing Puppet modules.
+
+{:.concept}
+## Installing the Puppet Development Kit
+
+Install the Puppet Development Kit (PDK) as your first step in creating and testing Puppet modules.
+
+The PDK package includes testing tools, a complete module skeleton, and command line tools to enable you to more easily create and test Puppet modules. PDK also includes all dependencies needed for its use.
+
+PDK includes the following tools:
+
+Tool   | Description
+----------------|-------------------------
+pdk | Command line tool for generating and testing modules
+rspec-puppet | Tests the behavior of Puppet when it compiles your manifests into a catalog of Puppet resources.
+puppet-lint | Checks your Puppet code against the recommendations in the Puppet Language style guide.
+puppet-syntax | Checks for correct syntax in Puppet manifests, templates, and Hiera YAML.
+metadata-json-lint | Validates and lints `metadata.json` files in modules against  Puppet's module metadatastyle guidelines.
+rspec-puppet-facts | Adds support for running rspec-puppet tests against the facts for your supported operating systems.
+puppetlabs_spec_helper | Provides classes, methods, and Rake tasks to help with spec testing Puppet code.
+
+PDK 1.0 is supported on:
+
+| Operating system | Version(s) | Arch | PkgType |
+| ---------------- | ---------- | ---- | ------- |
+| Red Hat Enterprise Linux | 6, 7 | x86_64 | RPM |
+| CentOS | 6, 7 | x86_64 | RPM |
+| Oracle Linux | 6, 7 | x86_64 | RPM |
+| Scientific Linux | 6, 7 | x86_64 | RPM |
+| SUSE Linux Enterprise Server | 11, 12 | x86_64 | N/A |
+| Ubuntu | 14.04. 16.04 | x86_64 | DEB |
+| Windows (Consumer OS) | 7, 8.1, 10 | x86_64 | MSI |
+| Windows Server (Server OS) | 2008r2, 2012, 2012r2, 2012r2 Core, and 2016 | x86_64 | MSI |
+| Mac OS X | 10.11, 10.12 | x86_64 | N/A |
+
+{:.section}
+### Before you begin
+
+PDK is a stand-alone development kit and does not require a pre-existing installation of Puppet. On Linux-based systems, you must enable the repository before you can download and install the package.
+
+If you used an early release version of PDK, we recommend you uninstall it before installing PDK 1.0. Use your platform's package manager to uninstall any PDK versions earlier than 1.0, and then install the updated 1.0 package.
+
+{:.task}
+## Enable the PDK repository on Linux
```

Comment:
Still waiting on more info from RE about our repository, if we get one at all. :(

For v1.0 there is a slight chance that we'd have to link to a download page and provide installation commands.